### PR TITLE
Added content for `avn service connection-pool`

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Added content for `avn service connection-pool`

### CODE REVIEW

**Comment by ftisiot** [comment]
Fixed #333

File: docs/tools/cli/service/connection-pool.rst [hunk 1]
```diff
@@ -31,17 +31,108 @@ Creates a new :doc:`PgBouncer connection pool </docs/products/postgresql/concept
...
+**Example:** In the service ``demo-pg`` Create a new connection pool named ``cp-analytics-it`` for the database ``it-analytics`` with:
 
 * username ``avnadmin``
 * pool-size of ``10`` connections 
 * ``transaction`` pool-mode
 
 ::
 
-  avn service connection-pool-create pg-doc \
+  avn service connection-pool-create demo-pg \
```
Comment (anelook) [inline docs/tools/cli/service/connection-pool.rst:42]: Interesting, when I'm trying to create  a connection pool with this command I get `ERROR	command failed: Error: {"errors":[{"message":"No valid password exists for user avnadmin.","status":400}],"message":"No valid password exists for user avnadmin."}`.

**Review by anelook — COMMENTED** [review]
I would either move, or add this article to PG docs (CLI or not). It is not a global functionality, but specific for Postres, right?

I couldn't run the commands, getting an error related to a password. Wanted to check with you what can be wrong and try it again.

**Comment by lornajane** [comment]
I'll chime in (but @anelook please keep driving this PR review) to say that I think all the CLI docs should stay together but I'd be very happy to have a per-product CLI cheatsheet, especially for specific commands like this one.

File: docs/tools/cli/service/connection-pool.rst (hunk 1 above)
Comment (ftisiot) [reply docs/tools/cli/service/connection-pool.rst:42]: This is weird, I just tried again and the command works as expected, maybe you can try again recreating the bug and open an issue in JIRA?

**Comment by ftisiot** [comment]
I'm with @lornajane, this is a command page and should stay together with the other commands. Yeah a cheatsheet would be good, should we open issues for this?

File: docs/tools/cli/service/connection-pool.rst (hunk 1 above)
Comment (anelook) [reply docs/tools/cli/service/connection-pool.rst:42]: Mystery, can't reproduce the error any more! Will keep an eye if I see it again

**Review by anelook — APPROVED** [review]
This time round all went smoothly! Tested commands without issues.

**Comment by anelook** [comment]
My opinion here is that the position of this command in commands tree is wrong, it belongs to PostreSQL specifically (can't be used with anything else, right?)

**Pull request merged**